.envrc

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Automatically sets up your devbox environment whenever you cd into this
+# directory via our direnv integration:
+
+eval "$(devbox generate direnv --print-envrc)"
```

Comment:
This is nice, but I don't think this should be the default for direnv, as I think more folks use direnv than they do devbox so the first thing they'd get greeted with is an odd error upon entering the project dir.

Happy to keep it as `.envrc-devbox` and add a note in the readme that a devbox setup is available and that optionally you can also use it automatically with direnv if you like.

Reply:
Ah yeah good point!
I'll rename the file to `.envrc-devbox` and update the docs too 👍